stylus/utilities/flexbox.styl

```diff
@@ -0,0 +1,197 @@
+@require '../settings/breakpoints'
+
+flexbox()
+    display flex
+
+inline-flexbox()
+    display inline-flex
+
+/* 1 Fix for Chrome 44 bug.
+ * https//code.google.com/p/chromium/issues/detail?id=506893 */
+flexbox-auto()
+    flex 1 1 auto
+    min-width 0 /* 1 */
+    min-height 0 /* 1 */
+
+flexbox-none()
+    flex none
+
+flexbox-column()
+    flex-direction column
+flexbox-row()
+    flex-direction row
+flexbox-wrap()
+    flex-wrap wrap
+flexbox-nowrap()
+    flex-wrap nowrap
+flexbox-wrap-reverse()
+    flex-wrap wrap-reverse
+flexbox-column-reverse()
+    flex-direction column-reverse
+flexbox-row-reverse()
+    flex-direction row-reverse
+
+flexbox-items-start()
+    align-items flex-start
+flexbox-items-end()
+    align-items flex-end
+flexbox-items-center()
+    align-items center
+flexbox-items-baseline()
+    align-items baseline
+flexbox-items-stretch()
+    align-items stretch
+
+flexbox-self-start()
+    align-self flex-start
+flexbox-self-end()
+    align-self flex-end
+flexbox-self-center()
+    align-self center
+flexbox-self-baseline()
+    align-self baseline
+flexbox-self-stretch()
+    align-self stretch
+
+flexbox-justify-start()
+    justify-content flex-start
+flexbox-justify-end()
+    justify-content flex-end
+flexbox-justify-center()
+    justify-content center
+flexbox-justify-between()
+    justify-content space-between
+flexbox-justify-around()
+    justify-content space-around
+
+flexbox-content-start()
+    align-content flex-start
+flexbox-content-end()
+    align-content flex-end
+flexbox-content-center()
+    align-content center
+flexbox-content-between()
+    align-content space-between
+flexbox-content-around()
+    align-content space-around
+flexbox-content-stretch()
+    align-content stretch
+
+flexbox-order-0()
+    order 0
+flexbox-order-1()
+    order 1
+flexbox-order-2()
+    order 2
+flexbox-order-3()
+    order 3
+flexbox-order-4()
+    order 4
+flexbox-order-5()
+    order 5
+flexbox-order-6()
+    order 6
+flexbox-order-7()
+    order 7
+flexbox-order-8()
+    order 8
+flexbox-order-last()
+    order 99999
+
+flexbox-grow-0()
+    flex-grow 0
+flexbox-grow-1()
+    flex-grow 1
+
+flexbox-shrink-0()
+    flex-shrink 0
+flexbox-shrink-1()
+    flex-shrink 1
+
+// Global classes generation
+
+breakpoints = {
+    'none': '',
+    'teeny': 'tt',
+    'tiny': 't',
+    'small': 's',
+    'medium': 'm',
+    'large': 'l',
+    'extra-large': 'xl'
+}
```

Comment:
What does none mean ?  When do we use it ?

I second bruno, why do we have so many breakpoints ?

Reply:
> What does none mean ? When do we use it ?

`none` means that there's no breakpoint to consider, in order to ouput the default class
```
.u-flex {
    display: flex;
}
```
> 
I second bruno, why do we have so many breakpoints ?

Because there is. I didn't add the breakpoints, it was already there. I just made sure there was a utility class working for every breakpoint we have in stock.